charts/kubescape-cloud-operator/values.yaml

```diff
@@ -413,3 +413,29 @@ registryScanScheduler:
 
   # Additional volumeMounts to be mounted on the scan scheduler
   volumeMounts: []
+
+# opentelemetry collector
+otelCollector:
+
+  # -- enable/disable metrics and traces collection
+  enabled: false
```

Comment:
[Question]: Maybe we should have this in the `values.yaml` file so users can override the default value